### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by marekful**
Minor improvements for email attachment and SD file download handling + a fix for when stopping a playback didn't finish cleanly.

### CODE REVIEW

File: mjpeg2sd.cpp [hunk 1]
```diff
@@ -585,6 +585,7 @@ void stopPlaying() {
   if (isPlaying) {
     // force stop any currently running playback
     stopPlayback = true;
+    controlFrameTimer(true);
```
Comment (marekful) [inline mjpeg2sd.cpp:588]: When `control?stopStream=1` is sent while an SD file playback is streaming, it almost always results in a ~10s delay before the playback could non-forcibly finish (and the HTTP request subsequently terminate). Note that I did these tests by replacing  the `while (isPlaying ...)` mechanism with waiting on a binary mutex which was released after clean playback finish.

Resetting the frame timer here seems to fix this but I haven't figured exactly what's going on. Would be glad to hear your opinion.

**Comment by s60sc** [comment]
Thanks, I'll need to reaquaint myself with the code first.

**Comment by s60sc** [comment]
Fix email header formatting when contains image attachment - cant see what this is fixing?

Add content-length header to browser download - good update, had to change: char contentLength[sizeof(uint32_t)]; to char contentLength[10]; to fit larger length strings

Reset controlFrameTimer when stopping playback - need to look at this

**Comment by s60sc** [comment]
Couldnt recreate SD playback problem

Added update for content-length header to browser download

**Pull request closed without merging**